rtf_vis_tool/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
```

Comment:
can we share this with the .gitignore in the top-level directory?

would eslintcache go in here?